sway/sway-input.5.scd

```diff
@@ -218,11 +218,16 @@ correct seat.
 	Set this seat as the fallback seat. A fallback seat will attach any device
 	not explicitly attached to another seat (similar to a "default" seat).
 
-*seat* <name> hide_cursor <timeout>
-	Hides the cursor image after the specified _timeout_ (in milliseconds)
-	has elapsed with no activity on that cursor. A timeout of 0 (default)
-	disables hiding the cursor. The minimal timeout is 100 and any value less
-	than that (aside from 0), will be increased to 100.
+*seat* <name> hide_cursor <timeout>|keyboard [enable|disable]
+	Hides the cursor image after the specified event occured.
+
+	If _timeout_ is specified, then the cursor will be hidden after _timeout_
+	(in milliseconds) has elapsed with no activity on the cursor. A timeout of 0
+	(default) disables hiding the cursor. The minimal timeout is 100 and any
+	value less than that (aside from 0), will be increased to 100.
+
+	If _keyboard_ is enabled, then the cursor will be hidden when typing
```

Comment:
What if I want it to hide when typing *or* after a timeout? I wonder if that should be supported.

Reply:
It is supported. The following configuration would cause the cursor to be hidden either when typing or after a 5 seconds timeout:
```
seat * hide_cursor keyboard enable
seat * hide_cursor 5000
```